cmd/cli/commands/orders.go

```diff
@@ -119,10 +120,19 @@ var orderCancelCmd = &cobra.Command{
 			return fmt.Errorf("cannot create client connection: %v", err)
 		}
 
-		orderID := args[0]
-		_, err = market.CancelOrder(ctx, &pb.ID{Id: orderID})
-		if err != nil {
-			return fmt.Errorf("cannot cancel order on Marketplace: %v", err)
+		request := &pb.OrderIDs{
+			Ids: make([]*pb.BigInt, 0, len(args)),
+		}
+		for _, idStr := range args {
```

Comment:
Great idea - decompose this. Then this will be oneliner. Also, there are several places where such boilerplate exists.

Reply:
not really oneliner, because go has no `map`